lib/main_development.dart

```diff
@@ -12,7 +13,7 @@ Future<void> main() async {
     final license = await rootBundle.loadString('assets/google_fonts/OFL.txt');
     yield LicenseEntryWithLineBreaks(['google_fonts'], license);
   });
-
+  NotificationService().initNotifications();
```

Comment:
Avoid duplicity, initialise in `lib/bootstrap.dart:52` like so:

`getIt<NotificationService>().initNotifications();` once you are done with the injection.